AVInput/AVInput.h

```diff
@@ -47,6 +47,8 @@ class AVInput: public PluginHost::IPlugin, public PluginHost::JSONRPC
 
     int m_primVolume;
     int m_inputVolume; //Player Volume
+
```

Comment:
update json file for documentaiton of new API's. Also if this plugin has unit test for L1 L2 make sure that is also coverred